package.json

```diff
@@ -125,6 +125,9 @@
         "zone.js": ">=0.14"
       }
     },
-    "overrides": {}
+    "overrides": {},
+    "onlyBuiltDependencies": [
```

Comment:
Why is this necessary?

Reply:
Can you please test what happens if you delete all node_modules along with all output folders in all packages (dist, www, loader, loaders, storybook-static, components-react/src/components/stencil-generated, icons/public/post-icons, etc.):

- [ ] Does the command `pnpm bootstrap` still work?
- [ ] Does the command `pnpm start` still work?
- [ ] Does the command `pnpm e2e` still work?
- [ ] Does the command `pnpm unit` still work?
- [ ] Does the command `pnpm lint` still work?